api.ts

```diff
@@ -588,7 +588,20 @@ export class CloudAPISDK {
         this.log('debug', `Database ${databaseId} ended up as '${database.status}' status after ${timePassedInSeconds}/${timeoutInSeconds} (Subscription ${subscriptionId})`);
         return database;
     }
-
```

Comment:
I'd add this back, I love the spacing between functions. maybe it more split and readable

Reply:
it splitted, instead of 2 lines space, 1 line now